Sources/Plasma/PubUtilLib/plGLight/plShadowMaster.cpp

```diff
@@ -91,7 +91,7 @@ void plShadowMaster::SetGlobalMaxSize(uint32_t s)
             if( (1 << i) & s )
                 break;
         }
-        s = 1 << i;
+        s = 1 << std::max(i, 0);
```

Comment:
> In practice, I'm pretty sure this won't happen and maybe an hsAssert is a better option here?

Yeah - if I'm reading the code correctly, `i` could only get down to -1 if `s` is 0, but in that case `((s-1) & ~s) != (s-1)` would be false, so we would never get here in the first place.

```suggestion
        ASSERT(i >= 0);
        s = 1 << i;
```